common/djangoapps/student/views.py

```diff
@@ -727,11 +727,11 @@ def dashboard(request):
         message for message in messages.get_messages(request) if 'account-activation' in message.tags
     ]
 
-    # Global staff can see what courses errored on their dashboard
+    # Global staff can see what courses encountered an error on their dashboard
```

Comment:
I know you're not going to like this, Jesse :D I did it before you told me to keep the diffs clean. Still mind it for this smaller commit?